Decred Wallet/en.lproj/Localizable.strings

```diff
@@ -383,3 +383,14 @@
 "errorMsg" = "Error Message";
 "info" = "Info";
 "delete" = "Delete";
+"allowOnce" = "Allow once";
+"alwaysAllow" = "Always allow";
+"notNow" = "Not now";
+
+/* Other Text */
+"send" = "send";
+"sent" = "sent";
+"receive" = "receive";
+"history" = "history";
+"settings" = "Settings";
+"help" = "help";
```

Comment:
As a way to prevent errors from creeping into our translation files, it's important to ensure that each line of code in the `LocalizedStrings.swift` file should find an equivalent declaration in **each/all** of the localizable files. Same comments, same order of declarations.

Reply:
resolved in aa27838

Reply:
moved these vars to the same location as in `LocalizedStrings.swift` in aa27838